cypress/fixtures/Certificate/CourseCertificate/credentials.json

```diff
@@ -2,5 +2,13 @@
   "free": {
     "email": "e2e-free-course-certificate@example.com",
     "password": "Abc12345-"
+  },
+  "standard": {
+    "email": "e2e-standard-course-certificate@example.com",
+    "password": "Abc12345-"
+  },
```

Comment:
Reuse this one please, fetch the testing database I generated the necessary data for this user.

Reply:
imo the test would be easier to implement if only had the data of unawarded stuff and not unrelated things but ok